### ISSUE TITLE
blocks that are chiseled into and chiseled bits place do not show

### ISSUE THREAD

**Issue by Sureman61** [ChiselsAndBits/Chisels-and-Bits#1270]
![image](https://github.com/user-attachments/assets/ef63d714-6e40-4dbc-ad7d-22690ebd5c2c)

**Comment by Sureman61** [comment]
This is on fabric 1.20.4 btw

**Comment by marchermans** [comment]
Do you have sodium installed?

**Comment by Joseph-Ghormley** [comment]
I download the mod and work fine with my server but my friend have the same issue where the block doesn,t show just the outline and yes we all have sodium

**Comment by Joseph-Ghormley** [comment]
ok, I figure out that I had Indium mod for distant horizon, and that why it work. my friend all add Indium and it work fine

**Comment by Sureman61** [comment]
> Do you have sodium installed?

yeah but i downloaded indium like @Joseph-Ghormley had their friends do it it works perfectly

**Comment by Joseph-Ghormley** [comment]
yes i did
